src/main/kotlin/org/cascadebot/cascadebot/data/objects/GuildData.kt

```diff
@@ -51,6 +50,9 @@ class GuildData(@field:Id val guildId: Long) {
 
     @Transient
     val pageCache = PageCache()
+
+    @Transient
+    val guildPermissionsManager = PerGuildPermissionsManager()
```

Comment:
Could we rename this to permissionsManager? Just because it's in guildData so I don't think we need another guild in the name